react/features/e2ee/middleware.js

```diff
@@ -38,7 +38,7 @@ MiddlewareRegistry.register(({ dispatch, getState }) => next => action => {
     case TOGGLE_E2EE: {
         const conference = getCurrentConference(getState);
 
-        if (conference) {
+        if (conference && conference.isE2EEEnabled() !== action.enabled) {
```

Comment:
Is this really needed? It is already checked internally...

Reply:
We also dispatch the sounds actions here, plus some other stuff, which we don't want unless enabled has changed.

Reply:
Makes sense!